src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Duke related Exceptions.
+ */
+public class DukeException extends Exception {
```

Comment:
Maybe you could extend `RuntimeException` instead of `Exception` to be more specific :)